github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -299,6 +299,45 @@ periodics:
         resources:
           requests:
             memory: "34Gi"
+- name: periodic-kubevirt-e2e-k8s-1.20-cgroupsv2
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "0 */8 * * *"
```

Let's fix the execution time so that we make sure we don't trigger a lot of jobs at the same time
```suggestion
  cron: "15 7,15,23 * * *"
```

---

sure. updated.